style.css

```diff
@@ -0,0 +1,300 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+  max-width: 960px; 
+  margin: auto;
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
+    width: 600px;
+  }
+  
+  nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  nav a {
+    text-decoration: none;
+    color: black;
+  }
+  
+  .blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+  }
+
+
+
+
+  .button {
+      background-color: #fb3e44;
+      color: white;
+  }
+
+  .secondSection {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: center;
+	align-items: center;
+    align-content: center;
+}
+
+.thirdSection{
+    display: flex;
+    justify-content: center;
+    padding: 40px;
+    
+}
+
+.box1{
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	justify-content: center;
+    padding: 20px;
+}
+
```

Comment:
Since classes b1-b4 all share the same padding, you could create one class `.box-size{padding: 20px;}` and apply it to the right corresponding `<div>` in your index.html